keymaps/templates/Apple Mail.cson

```diff
@@ -4,16 +4,39 @@
 # http://dashkards.com/img/dashkard-mail.png
 
 'body':
+
+  ### Jumping ###
+  'cmd-1' : 'navigation:go-to-inbox'
```

Comment:
Can we make these `cmd-shift-1` since cmd-1 is bound to "switch to account 1"? I don't think we can override those unfortunately.